src/data/linux-quiz.ts

```diff
@@ -841,6 +841,16 @@ const linuxQuiz = [
       "KDE is a popular desktop environment which resembles the Windows user interface.",
     Link: "https://www.freecodecamp.org/news/linux-basics/",
   },
+  {
+    Question: "Which of the following is NOT a category of file permissions in Linux?",
+    Answer: "compile",
+    Distractor1: "execute",
+    Distractor2: "write",
+    Distractor3: "read",
+    Explanation:
+      "In Linux, file permissions fall into three categories: read, write and execute.",
+    Link: "https://www.freecodecamp.org/news/linux-chmod-chown-change-file-permissions/",
+  },
```

Comment:
This is a good addition.

Can you also remove this same question from the `security` quiz. 
Unfortunately it was miscategorized when we split up the quiz files.

You will probably find it closer to the top of the `security` quiz file. 👍

Reply:
Sure , I will. @jdwilkin4